test/commands/pull.js

```diff
@@ -0,0 +1,162 @@
+                },
+                {
+                    name: 'requirements.txt',
+                    format: 'TEXT',
+                    content: '# Add your dependencies here.\n# See https://pip.pypa.io/en/latest/reference/requirements-file-format/\n# for how to format them\napify ~= 1.0.0\n',
+                },
+            ],
+        },
+    ],
+};
+
+const TEST_ACTOR_GITHUB_GIST = {
+    name: ACTOR_NAME,
+    isPublic: false,
+    versions: [
+        {
+            versionNumber: '0.0',
+            sourceType: 'GITHUB_GIST',
+            buildTag: 'latest',
+            gitHubGistUrl: 'https://gist.github.com/DennisKallerhoff/32f1efd686e11f6a05f1af87bddb1f1a',
+        },
+    ],
+};
+
+const TEST_ACTOR_GIT_REPO = {
+    name: ACTOR_NAME,
+    isPublic: false,
+    versions: [
+        {
+            versionNumber: '0.0',
+            sourceType: 'GIT_REPO',
+            buildTag: 'latest',
+            gitRepoUrl: 'https://github.com/HonzaTuron/baidu-scraper#multipage',
+        },
+    ],
+};
+
+describe('apify pull', () => {
+    before(async function () {
+        if (fs.existsSync(GLOBAL_CONFIGS_FOLDER)) {
+            // Skip tests if user used CLI on local, it can break local environment!
+            console.warn(`Test was skipped as directory ${GLOBAL_CONFIGS_FOLDER} exists!`);
+            this.skip();
+            return;
+        }
+        await command.run(['login', '--token', TEST_USER_TOKEN]);
+        process.chdir(ACTOR_NAME);
+    });
+
+    beforeEach(() => {
+        sinon.spy(console, 'log');
+    });
+
+    it('should fail outside actor folder without actorId defined', async () => {
+        try {
+            await command.run(['pull']);
+        } catch (err) {
+            expect(err.message).to.be.eql('Cannot find actor in this directory.');
+        }
+    });
+
+    it('should work with actor SOURCE_FILES', async () => {
+        const testActor = await testUserClient.actors().create(TEST_ACTOR_SOURCE_FILES);
```

Comment:
It would be good to delete all of these created actors in an `afterAll` method, so that they don't accumulate on the test user.